fenix/app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -211,6 +214,26 @@ class BrowserFragment : BaseBrowserFragment(), UserInteractionHandler {
             owner = viewLifecycleOwner,
             view = binding.root,
         )
+
+        setTranslationFragmentResultListener()
+    }
+
+    private fun setTranslationFragmentResultListener() {
```

Comment:
I was wondering if we were able to just show the snackbar from within the bottom sheet fragment like https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/java/org/mozilla/fenix/onboarding/WallpaperOnboardingDialogFragment.kt#132?

Just wondering if we could reduce the amount of code handling with emitting fragment result to show this snackbar if there is another way to do this.

Reply:
It doesn't work. I tried this. I need to show the snackbar in onDismiss.